EIPS/eip-gas_to_eth.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
```suggestion
---
eip: 7791
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.